realsense2_camera/src/ros_sensor.cpp

```diff
@@ -153,7 +153,7 @@ void RosSensor::set_sensor_parameter_to_ros(rs2_option option)
 {
     std::string module_name = create_graph_resource_name(rs2_to_ros(get_info(RS2_CAMERA_INFO_NAME)));
     const std::string option_name(module_name + "." + create_graph_resource_name(rs2_option_to_string(option)));
-    float value = get_option(option);
+    int value = get_option(option);
```

Comment:
Can you explain how is this solve the issue?
I we do need it I suggest going with
‘auto value = static_cast<int>(get_option(option))’

Reply:
What about float options? All will be converted to int?
Exposure for example?